recipes/nanovar/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "nanovar" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 806e5efa90ae7f06f7848e66eb82149924040ec8773b4f0c73738252a25ec23c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - biopython ==1.74
+    - keras ==2.2.4
+    - matplotlib ==2.2.3
+    - natsort ==6.2.0
+    - numpy ==1.17.3
+    - progress ==1.4
+    - pybedtools ==0.8.0
+    - python >=3.5
+    - scipy ==1.2.1
+    - tensorflow ==1.13.1
```

Comment:
Do we really need to ping to these exact versions, rather than these being minimum versions?

Reply:
I encountered conflicts/error when I do that. Let me try again now.

Reply:
I have resolved the error. Version 1.2.2 allows those dependencies being minimum versions. Should make a new commit?

Reply:
Yes please!